### PR TITLE
Added security contact

### PR DESCRIPTION
**Pull request by ajinkya1986**

### CODE REVIEW

File: cartography/intel/azure/securitycenter.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    try:
+        security_contacts = list(map(lambda x:x.as_dict(), client.security_contacts.list()))
+        return security_contacts
+
+    except HttpResponseError as e:
+        logger.warning(f"Error while retrieving security contacts - {e}")
+        return []
+
+@timeit
+def transform_security_contacts(security_contacts: List[Dict], subscription_id: str, common_job_parameters: str):
```
Comment (ganeshjagdale9545) [inline cartography/intel/azure/securitycenter.py:35]: Return format for method is missing

File: cartography/intel/azure/securitycenter.py [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+from cartography.util import run_cleanup_job
+from cartography.util import timeit
+
+logger = logging.getLogger(__name__)
+azure_console_link = AzureLinker()
+
+def load_security_contacts(session: neo4j.Session, subscription_id: str, data_list: List[Dict], update_tag: int) -> None:
+    session.write_transaction(_load_security_contacts_tx, subscription_id, data_list, update_tag)
+
+def get_security_center_client(credentials: Credentials, subscription_id: str, region: str):
```
Comment (ganeshjagdale9545) [inline cartography/intel/azure/securitycenter.py:20]: Return format for method is missing

File: cartography/intel/azure/securitycenter.py [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+    tx: neo4j.Transaction, subscription_id: str, security_contacts: List[Dict], update_tag: int,
+) -> None:
+    ingest_contacts = """
+    UNWIND {CONTACTS} as cont
+    MERGE (c:AzureSecurityContact{id: cont.id})
+    ON CREATE SET c.firstseen = timestamp(),
+    c.name = cont.name,
+    c.consolelink = cont.consolelink,
+    c.phone = cont.phone
+    SET c.lastupdated = {update_tag}
```
Comment (ganeshjagdale9545) [inline cartography/intel/azure/securitycenter.py:58]: region property missing

File: cartography/intel/azure/securitycenter.py [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+    neo4j_session: neo4j.Session, credentials: Credentials, subscription_id: str, update_tag: int,
+    common_job_parameters: Dict, regions: List
+) -> None:
+
+    for region in regions:
+        client = get_security_center_client(credentials, subscription_id, region)
+        contacts = get_security_contacts_list(client)
+        security_contacts_list = transform_security_contacts(contacts, subscription_id, common_job_parameters)
+
+        if common_job_parameters.get('pagination', {}).get('security_contacts', None):
```
Comment (ganeshjagdale9545) [inline cartography/intel/azure/securitycenter.py:85]: `security_contacts` change to `securitycenter `

**Review by ganeshjagdale9545 — CHANGES_REQUESTED** [review]
minor changes needed

**Review by ganeshjagdale9545 — APPROVED** [review]
looks good

**Review by mpurusottamc — APPROVED** [review]
looks good to me.

**Pull request merged**